output/src/OutputTemplateConverterStrategy.ts

```diff
@@ -1,9 +1,220 @@
-import { OutputTemplate } from '.';
+import _defaultsDeep from 'lodash.defaultsdeep';
+import { PartialDeep } from 'type-fest';
+import {
+  Carousel,
+  DynamicEntities,
+  MessageValue,
+  OutputTemplate,
+  OutputTemplateBase,
+  OutputTemplatePlatforms,
+  plainToClass,
+  QuickReplyValue,
+} from '.';
 
-export interface OutputTemplateConverterStrategy<RESPONSE extends Record<string, unknown>> {
-  responseClass: new () => RESPONSE;
+export interface SanitizationConfig {
+  // truncate arrays that exceed maximum length
+  maxArraySize: boolean;
```

Comment:
With this property name, I would assume a number. Should we maybe rename both?

Since `sanitization` could do more than truncating things, maybe we could add `truncate` or `trim`to the properties?
* `truncateToMaxArraySize`, `truncateArrays`, `trimToMaxArraySize`, `trimArrays`
* `truncateToMaxStringLength`,`truncateStrings`, `trimToMaxStringLength`, `trimStrings`